TRAIN.md

```diff
@@ -199,7 +199,7 @@ python main.py --mode run-relaxations --config-yml configs/s2ef/2M/schnet/schnet
 ```
 The relaxed structure positions are stored in `[RESULTS_DIR]/relaxed_positions.npz` and later used to create a submission file to be uploaded to EvalAI. Predicted trajectories are stored in `trajectories` directory for those interested in analyzing the complete relaxation trajectory.
 
-## Create EvalAI submission files
+## Create EvalAI OC20 submission files
```

Comment:
This breaks hyperlinks at the top of the page, see [this](https://github.com/Open-Catalyst-Project/ocp/blob/649c4df3e411cf367cbb5a3ccf55757806250a55/TRAIN.md).

Reply:
Also the title of the page is currently OC20-focused, might want to make a separate top-level OC22 section.